core/src/main/python/wlsdeploy/tool/create/domain_creator.py

```diff
@@ -167,6 +169,7 @@ def create(self):
         self.__fail_mt_1221_domain_creation()
         self.__create_domain()
         self.__deploy()
+        self.__create_boot_dot_properties()
```

Comment:
Why was this added back?  Shouldn't this be removed?

Reply:
verbally discussed.  boot.properties is only created when startup is set to development mode (not in production mode).  This is the same behavior that the WLS config tool uses.